### ISSUE TITLE
Take a close looks at categories and place actions in the correct section

### ISSUE THREAD

**Issue by jessicalostinspace** [sdras/awesome-actions#349]
I have reorganized the sections previously and added several sections including the Static Analysis section but I think there needs to be a closer look into appropriately placing the some of the actions in "Testing" to make sure they are correctly placed there. There are several other areas like "Third Party" that have actions in them that would more appropriately be placed elsewhere. I will be happy to look into this tonight. If you have any other suggestions, please feel free to discuss here :)

**Comment by fabasoad** [comment]
Good idea. Also, I think we need to create "Setup actions" category as well (like we have for Official Actions - https://github.com/sdras/awesome-actions#setup-actions). So, we can move there "Setup Xamarin", "Setup Mint", "Setup Brainfuck", "Setup NativeScript". In a future we can create a subcategory there if needed.

**Comment by sdras** [comment]
I'm open to a PR for this, as long as there are more than 3 items in a new subcategory.
